demos/src/demo.scss

```diff
@@ -7,7 +7,11 @@ body {
 
 h6 {
 	@include oTypographyProductHeadingLevel6();
-	margin-top: 20px;
+	margin: 20px;
+
+	&.center {
```

Comment:
`.center` appears to be no longer used 🎉

Reply:
totally is, in `alert-inner.mustache` and `notice-inner.mustache`

Reply:
Ohhhh. 👍